src/MvvmNano.Forms/BetterMasterDetailPage.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Linq; 
+using Xamarin.Forms;
+
+namespace MvvmNano.Forms
+{
+    /// <summary>
+    /// A MasterDetailPage implementation that fits to the MvvmNano framework.
+    /// </summary>
+    public class BetterMasterDetailPage :  MasterDetailPage
```

Comment:
You saw that i created a MvvmNanoMasterDetailPage<> that inherits BetterMasterDetailPage aswell to avoid casting the generic MvvmNanoMasterDetailPage<> in the MvvmNanoApplication 
->  public BetterMasterDetailPage MasterPage => MainPage as BetterMasterDetailPage;